Example-iOS/Source/Examples/SwiftUI/SwiftTouchEvents.swift

```diff
@@ -37,7 +37,7 @@ struct SwiftTouchEvents: DismissableView {
                 bearGuy.view()
                     .aspectRatio(1, contentMode: .fit)
 
-                clock.controlsView()
+                clock.view()
```

Comment:
This is overriding the base .view() method rather than making a new one. It wasn't possible before using opaque return values (ie `func foo() -> some View`)